main.c

```diff
@@ -145,6 +145,7 @@ open_by_handle_at (int mount_fd, struct file_handle *handle, int flags)
 #define PRIVILEGED_OPAQUE_XATTR "trusted.overlay.opaque"
 #define PRIVILEGED_ORIGIN_XATTR "trusted.overlay.origin"
 #define OPAQUE_WHITEOUT ".wh..wh..opq"
+#define WHITEOUT_MAX_LEN (sizeof (OPAQUE_WHITEOUT))
```

Comment:
Note that eats an extra character, since sizeof also counts the trailing NUL.

Reply:
yeah, fortunately I've found out the bug before the PR was merged and fixed it in the last version (https://github.com/containers/fuse-overlayfs/pull/237/files#diff-2045016cb90d1e65d71c2407a2570927R148).  Another issue is that the opaque whiteout is not used as a prefix, so it is enough to account for `.wh.`